app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java

```diff
@@ -89,23 +90,30 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         if (savedInstanceState != null) {
             tag = savedInstanceState.getParcelable(TAG_MODEL);
         }
+        editActivityHelper = new EditActivityHelper(this);
         prepareLocationHandling();
     }
 
+    @Override
+    protected void onDestroy() {
```

Comment:
onDestroy() is not a good idea, its not called every time, much better is onStop()